src/client/components/pages/collection.js

```diff
@@ -163,45 +171,83 @@ class CollectionPage extends React.Component {
 			request.post(submissionUrl)
 				.send({bbids})
 				.then((res) => {
-					window.location.href = `/collection/${this.props.collection.id}`;
+					this.setState({
+						message: {
+							text: `Removed ${bbids.length} ${_.kebabCase(this.props.collection.entityType)}
+							${bbids.length > 1 ? 's' : ''}`,
```

Comment:
SomehowBecause this is on a new line, it ends up with a space between entity type and 's':

<img width="233" alt="Capture d’écran 2020-08-13 à 13 00 13" src="https://user-images.githubusercontent.com/6179856/90127253-2e831780-dd65-11ea-8e88-0ff97dcb0005.png">

This is the only place it does that though, the other messages are fine.